docs/root/intro/arch_overview/outlier.rst

```diff
@@ -113,7 +146,7 @@ action
 
 type
   If ``action`` is ``eject``, specifies the type of ejection that took place. Currently type can
-  be one of ``5xx``, ``GatewayFailure`` or ``SuccessRate``.
+  be one of ``5xx``, ``GatewayFailure``, ``SuccessRate``, ``ConnectFailure`` or ``MalformedPayload``.
```

Comment:
What is "MalformedPayload" ?

Reply:
My bad. Thanks for catching it. This is how I originally called "upstream server failure", but later on changed naming.